Additions/UIView-KIFAdditions.m

```diff
@@ -458,7 +459,7 @@ - (void)tapAtPoint:(CGPoint)point;
     
     if ([NSStringFromClass([self class]) isEqual:@"UIWebBrowserView"]) {
         webBrowserView = self;
-    } else if ([self isKindOfClass:[UIWebView class]]) {
+    } else if ([self isKindOfClass:[WKWebView class]]) {
```

Comment:
Does it make sense to maintain some `UIWebView` support side-by-side until Apple starts rejecting app store submissions (I assume will likely be in the Spring)?

Reply:
Hi @justinseanmartin 
WKWebview is available since iOS 8 like KIF project. Sooner people has to move to WKWebview and they can because is supported.

We can keep both something like:
`else if ([self isKindOfClass:[UIWebView class]] || [self isKindOfClass:[WKWebView class]])`
But we will continue receiving the Apple warning on submission.

Reply:
I'm thinking from a backwards compatibility perspective for projects that still rely on UIWebView. We shouldn't break the test framework from working on those projects while it is still allowed for app submissions.

Why are you linking KIF into your app for app store build? The KIF code generally shouldn't be present in Release artifacts, because it uses a bunch of private APIs. If you're using CocoaPods, you can use the pod `configuration` option (see [Podfile syntax](https://guides.cocoapods.org/syntax/podfile.html#pod)) to restrict it to only being linked in Debug builds for example.